program_info/CMakeLists.txt

```diff
@@ -41,6 +41,21 @@ configure_file(org.prismlauncher.PrismLauncher.metainfo.xml.in org.prismlauncher
 configure_file(prismlauncher.rc.in prismlauncher.rc @ONLY)
 configure_file(prismlauncher.manifest.in prismlauncher.manifest @ONLY)
 configure_file(prismlauncher.ico prismlauncher.ico COPYONLY)
+
+if(MSVC)
+    set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP true)
+    include(InstallRequiredSystemLibraries)
```

Comment:
Why are these set twice?

Reply:
I mostly was thinking because there were two different files and I'm not completely sure which one will run first that setting them in both places would just be easier

Reply:
We can set `CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP` in the subdirectory as long as we use the `PARENT_SCOPE` option with it, and that will propagate to when it's used after the `add_subdirectory()` call in the root file

We can't do that with `include(`) though, so it'll still be imported twice; it would probably be best to just include it in the root instead prior to either of these

Reply:
ok, this what you intended?